gluoncv/data/otb/tracking.py

```diff
@@ -0,0 +1,212 @@
+"""Visual Tracker Benchmark."""
+import json
+import os
+import cv2
+from tqdm import tqdm
+
+class Dataset(object):
+    """Dataset."""
+    def __init__(self, name, dataset_root):
+        self.name = name
+        self.dataset_root = dataset_root
+        self.videos = None
+
+    def __getitem__(self, idx):
+        if isinstance(idx, str):
+            return self.videos[idx]
+        elif isinstance(idx, int):
+            return self.videos[sorted(list(self.videos.keys()))[idx]]
+        return None
+
+    def __len__(self):
+        return len(self.videos)
+
+    def __iter__(self):
+        keys = sorted(list(self.videos.keys()))
+        for key in keys:
+            yield self.videos[key]
+    def set_tracker(self, path, tracker_names):
+        """
+        Args:
+            path: path to tracker results,
+            tracker_names: list of tracker name
+        """
+        self.tracker_path = path
+        self.tracker_names = tracker_names
+
+class Video(object):
+    """
+    Args:
```

Comment:
Please describe what is this class using 1 or 2 sentences. And please provide more explanations for arguments as well.